codespell_lib/data/dictionary.txt

```diff
@@ -12858,6 +12939,7 @@ efford->effort, afford,
 effords->efforts, affords,
 effulence->effluence
 eforceable->enforceable
+efore->before
```

Comment:
`afore`?

I would like to be able to add some fixes in a dictionary of rare fixes...

Reply:
> `afore`?

Added.
 
> I would like to be able to add some fixes in a dictionary of rare fixes...

Yup, that would be useful.